index.html

```diff
@@ -76,127 +127,131 @@ <h1 class="projects-title">My Recent Works</h1>
+                  <span class="tech">bootstrap</span>
+                  <span class="tech">Ruby</span>
+                </div>
               </div>
-            </div>
-            <div class="card-variant-footer">
-              <button class="btn btn-block">See Project</button>
-            </div>
-          </article>
+              <div class="card-variant-footer">
+                <button class="btn btn-block">See Project</button>
+              </div>
+            </article>
+            <article class="project-card-variant">
+              <div class="card-variant-body">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-info">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard
+                </p>
+                <div class="tech-container">
+                  <span class="tech">html</span>
+                  <span class="tech">bootstrap</span>
+                  <span class="tech">Ruby</span>
+                </div>
+              </div>
+              <div class="card-variant-footer">
+                <button class="btn btn-block">See Project</button>
+              </div>
+            </article>
+            <article class="project-card-variant">
+              <div class="card-variant-body">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-info">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard
+                </p>
+                <div class="tech-container">
+                  <span class="tech">html</span>
+                  <span class="tech">bootstrap</span>
+                  <span class="tech">Ruby</span>
+                </div>
+              </div>
+              <div class="card-variant-footer">
+                <button class="btn btn-block">See Project</button>
+              </div>
+            </article>
+            <article class="project-card-variant">
+              <div class="card-variant-body">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-info">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard
+                </p>
+                <div class="tech-container">
+                  <span class="tech">html</span>
+                  <span class="tech">bootstrap</span>
+                  <span class="tech">Ruby</span>
+                </div>
+              </div>
+              <div class="card-variant-footer">
+                <button class="btn btn-block">See Project</button>
+              </div>
+            </article>
+          </div>
         </div>
       </section>
       <section class="about-section">
```

Comment:
- [ ] Your about section is good, but there is a gray line dividing the about section and the section containing the skills. Kindly add this line to make your desktop version like the template.
 - Template design:
 - 
<img width="899" alt="image" src="https://user-images.githubusercontent.com/61919617/206176749-cc7f8705-f852-4e22-b1d2-460b2dab790e.png">